plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -522,27 +522,6 @@ def make_template(in_str):
         for k, v in save_ephemeral_nodes.items():
             if getattr(self.dbt_manifest.nodes[k], "compiled", False):
                 self.dbt_manifest.nodes[k] = v
-
```

Comment:
This hacky code is no longer needed. We now pass the correct source string to `slice_file()`, so the results that come back are already correct.